CI/include/build_support.sh

```diff
@@ -57,9 +57,22 @@ BUILD_CONFIG="${BUILD_CONFIG:-RelWithDebInfo}"
 CI_WORKFLOW="${CHECKOUT_DIR}/.github/workflows/main.yml"
 CURRENT_ARCH="$(uname -m)"
 CURRENT_DATE="$(date +"%Y-%m-%d")"
+GIT_VERSION=""
 
 ## Utility functions ##
 
+is_gte() {
```

Comment:
I'm inclined to leave this in-place in case we find another use for it somewhere else.

Reply:
I'd rather follow YAGNI here, but if we want to make this multi-purpose, I'd rather have a `check_version` function that takes 3 arguments (1. value, comparator, 2. value) and implement the different comparisons there (equality/non-equality is simple, whereas `gte` and `lte`  just change a single parameter to `sort` or changes `head` for `tail`).

Also use return codes instead of strings, that's very "unshell"-ish.

Reply:
For a future iteration, I think that would be a good change.  For this iteration, I wrote what I needed without building a full set of comparison functions/operations.

Either way, I'll mull this over for a bit for:
1. whether this remains a separate function or if I merge it into `github_fetch`, though I'd probably lean towards the former
2. whether this will be implemented now or pushed till a later iteration (leaning towards the latter)